calico-policies/private-network-isolation/calico-v3/au-syd/allow-private-services-pods.yaml

```diff
@@ -70,6 +70,21 @@ spec:
       - 514
     protocol: UDP
     source: {}
+  - action: Allow
```

Comment:
What is Atracker and what would IKS/ROKS customers use it for?  Is it needed for all customers, or just a (small or large) subset of customers?  Or just supported on classic or VPC?

Reply:
I think this means Activity Tracker,  but if so, I have a secondary question - I was under the impression that activity tracker uses the same endpoints as log analysis?

Reply:
As of now, It is only for classic IKS customers. We are planning to have a GA in August,2021.

Reply:
@bradbehle @Rachael-Graham